### DISCUSSION TITLE
not showing arrows

### DISCUSSION THREAD

**Discussion by SilvaLucas92** [pierpo/react-archer#191] [Q&A]
Hi ! how are you! I want to ask you I'm using react archer with a scrolContainer(react-indiana-drag) and when I reach the bottom of the container the arrows dissapear. any idea why it is happening? thanks!

**Comment by pierpo** [comment]
Hey! You should put some screenshots ;)
If you inspect your view, it might mean that some container is not big enough. Try adding some padding at the bottom of your container :thinking:

**Comment by Aravind742001** [comment]
Hi guys! I'm currently using react archer and I've displayed the arrows for my data. now I need to keep a hover functionality for the arrows to disappear when I hover the mouse on the arrows. Is there any functions for this within the package ?

**Reply by pierpo** [reply]
Hey! This could've been a new discussion haha

It's a feature that I implemented but never merged here https://github.com/pierpo/react-archer/pull/183

I should go back on it, I think it was mostly ready! I don't remember why I didn't merge haha

**Reply by Aravind741** [reply]
@pierpo can you please merge that, because it will be helpful to complete my table.
